### PR TITLE
Feature/34

### PR DESCRIPTION
**Pull request by ayushj117**

### CODE REVIEW

**Comment by manish-nx** [comment]
Please resolve all the conflicts before.

**Comment by ayushj117** [comment]
I resolved the conflicts

File: expressEngine/product/parser/parser.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// import base Parser
+import {
+  BaseParser,
+} from '../../lib/parser/index';
```
Comment (aman-successive) [inline expressEngine/product/parser/parser.js]: Remove index from import statement

File: expressEngine/product/service/service.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import { BaseService, DBService } from '../../lib/service/index';
 
 // import collections
-import { ProductModel, ProductDetail } from '../../model/index';
+import { Product, ProductDetail } from '../../model/index';
```
Comment (aman-successive) [inline expressEngine/product/service/service.js]: remove index from import statement

File: expressEngine/user/parser/parser.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// import the controller
+import { UserController } from '../controller/index';
+
+// import base controller
+import { BaseParser } from '../../lib/parser/index';
```
Comment (aman-successive) [inline expressEngine/user/parser/parser.js]: remove index from import

**Comment by NishantNaithani1997** [comment]
@ayushj117 Please update your code with the develop branch.
I request all NOT TO MERGE this PR.

**Comment by yogeshsinghh** [comment]
please add description or link of the issue no

**Comment by mayank-nx** [comment]
https://github.com/nodexpertsdev/RnD/pull/88

**Pull request closed without merging**